CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+Let's denote this the **topic** folder.
+
+Now, look inside the **topic**. You will see a list of folders, one for each tutorial.
+Check that the tutorial you want to create hasn't been created yet.
+It may also be the case to check whether there is a *Pull Request* open.
+Otherwise, decide how you're going to call your tutorial -- let's denote it the **title** of your tutorial.
+For organization, both **topic** and **title** need to be *lowercase* and *hyphen-separated*.
+Now that you decided the **topic** and **title** of your tutorial, it's time to create it.
+
+First, *fork* the JSOTutorials.jl repository to your account.
+This should create a repository of the form `https://github.com/USER/JSOTutorials.jl`.
+Notice the **USER**, -- it should be you! -- we will use it later.
+In your computer, clone your repository, and enter the folders **topic** and **title**, creating them if necessary.
+Inside this folder you will start a [Weave.jl](https://github.com/JunoLab/Weave.jl) file: a file with extension `.jmd`.
+If you're not familar with Weave, it may be easier to create a Jupyter Notebook instead, and later convert it to `.jmd`, or to use Atom with specific packages to render the `.jmd` file.
+Please refer to the [Weave.jl](https://github.com/JunoLab/Weave.jl) tutorial for more information.
+Either way, I will assume you now created `.jmd` files.
+
+It's imperative that you create a file called **title.jmd**, i.e., the file should have the same name as the folder. The header of the file is something like
+```yaml
+---
+title: Some Title
+author: Your Name
+---
+
+![](../../jso.png)
+```
+Although `Some Title` doesn't have to be exactly  the same as **title**, it's better to keep it close.
+Notice that the logo `jso.png` is included as well.
+It's also important that the tutorial is reproducible, so you need to create a environment and keep the `Project.toml` and `Manifest.toml` files.
+I recommend something like
+```julia
+using Pkg
+pkg"activate ."
+if false # Install these packages:
+  pkg"add THEPACKAGESYOUNEED"
+end
+pkg"instantiate"
+```
+This part might be improved in the future, but for now it shows how to install the packages, activates the environment and instantiates it.
+
+Work on it until you are satisfied with your tutorial.
+Now you should have a new file **title.jmd** inside folder **title** inside of **topic**.
+To submit your tutorial, you will need a branch and you will need to create the various formats that we support for our tutorials.
+Then, inside **topic/title**, enter
+```julia
+pkg> activate ../..
+julia> using JSOTutorials
+julia> JSOTutorials.conversions("title.jmd")
+pkg> test
+```
+This should create `.html`, `.ipynb` and `.jl` files using `Weave.jl`, and then test that they exits.
+
+Now comes the online part. First, go to [Travis-CI](https://travis-ci.org/) and enable travis for `JSOTutorials.jl`.
+Then, go to your github and create a [personal token](https://github.com/settings/tokens).
```

Comment:
You should also mention that in the token scope, the ```repo``` flag needs to be selected, to allow travis to have full access of the repository.

Reply:
Thanks, I've updated it.